### DISCUSSION TITLE
windows 版本0.3.2 机型redmi book 14 （系统win11，64位）无法录制声音

### DISCUSSION THREAD

**Discussion by z-wddmx** [ZuodaoTech/everyone-can-use-english#871] [General]
![Uploading 微信图片_20240723103646.png…](
![微信图片_20240723103646](https://github.com/user-attachments/assets/e8e7550c-16b2-47da-824c-ff1820ab12df)
)

**Comment by an-lee** [comment]
图片没上传成功。复制图片之后，等它上传完成之后再点发布。

**Reply by z-wddmx** [reply]
图片传不上来，报错failedToSaveRecording：Error invocationing remote method 'recordings-create'：Error：未检测到任何声音

**Reply by z-wddmx** [reply]
已经解决了，是我声音输入设置成立体声混响导致的，改回原来默认的麦克风列阵就可以了

**Comment by z-wddmx** [comment]
图片传不上来，报错failedToSaveRecording:Error invoking remote method 'recordings-create':Error:未检测到任何声音
